DESCRIPTION.md

```diff
@@ -0,0 +1,33 @@
+LMCTL is a command-line client that provides commands for interacting with IBM Telco Network Cloud Manager (TNCM) Orchestrator (TNCO/ALM) environments. 
+
+In addition it includes an opinionated pattern for managing xNF/Network Service designs during the CICD lifecycle, as file based projects, to produce packages suitable for production release.
+
+## Quick Install
+
+Assumes you already have Python3.6+ and have decided if LMCTL should be installed in a [virtual environment](https://pypi.org/project/virtualenv/) (recommended) or globallly.
```

Comment:
Think we need another 'l' in globally. Or one less, I'm not sure

Reply:
More L's. Got it.